examples/nodejs/decrypt.mjs

```diff
@@ -17,23 +17,26 @@ process.removeAllListeners('warning'); // To remove experimental fetch warnings
     const encryptedDataHexEncodedIndex = process.argv.indexOf('--encryptedDataHexEncoded') + 1;
```

Comment:
all mjs files seem to be double indented (4 spaces instead of 2)

Reply:
Done in 20da090